modules/nixos/hdr.nix

```diff
@@ -19,8 +19,8 @@ let
 
   configuration = strength: {
     boot.kernelPackages = strength cfg.kernelPackages;
-    programs.steam.gamescopeSession = {
-      enable = true; # HDR can't be used with other WM right now...
+    programs.steam.gamescopeSession.enable = true; # HDR can only be used with headless Gamescope right now...
```

Comment:
I don't think headless is the right word here, that's for when there is no output attached. It should be "rendering directly on TTY/KSMDRM/DRI or headless".

But nonetheless, looks good to me.